package-lock.json

```diff
@@ -1,627 +0,0 @@
-{
```

Comment:
Why did you delete this file? Don't we usually keep it around for dependency history?

Reply:
We have a yarn.lock now. Going to try out yarn for this repo